src/content.js

```diff
@@ -22,6 +22,7 @@ import linkifyCode, {editTextNodes} from './libs/linkify-urls-in-code';
 import autoLoadMoreNews from './libs/auto-load-more-news';
 import addOPLabels from './libs/op-labels';
 import addReleasesTab from './libs/add-releases-tab';
+import removeUploadFilesButton from './libs/remove-upload-files-button';
```

Comment:
Can you also move this file to the new `features` folder? It was created in #793 and completed by #796